rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -457,7 +457,9 @@ setup:
               range:
                 field: date
                 ranges:
-                  { from: 2021-05-01T00:00:00Z, to: 2021-05-05T00:00:00Z }
+                 -
+                   from: 2021-05-01T00:00:00Z
+                   to: 2021-05-05T00:00:00Z
```

Comment:
I think you can also quote these and they should work in both cases. I know I sort of habitually quote these.

It'd be nice if we could detect cases where this'd break the client in our build or test infrastructure.

Reply:
I also wondered about that but found an example using this form which I believe works now. I agree, a check to ensure that the client parsers don't break when new tests come in would be nice.